openpype/settings/defaults/system_settings/applications.json

```
@@ -835,6 +835,11 @@
                 "linux": "/opt/Python/3.6/bin"
             }
         },
+        "copy_fusion_settings": {
+            "copy_prefs": false,
+            "prefs_path": "~/.openpype/hosts/fusion/prefs",
```

This should be maybe empty by default.
```suggestion
            "prefs_path": "",
```

---

I thought about this string as a default value that would work out of the box on any host. Say, first setup, check the `Copy` checkbox, boom, now you have working preferences already. But probably I should work more to the @EmberLightVFX suggested way, and set the prefs location project-wise...

---

It could place the preference file in a folder named after the project by default. That way nothing will be overwritten by different project.

---

I'd prefer this in Project Settings instead. We'd likely have this value across all projects but it'd be very nice for a particular project to have very customized Fusion prefs on its own.

The path itself seems like a fine default path too me.